ImageDownloader.py

```diff
@@ -2,11 +2,16 @@
 
 import urllib
 import os
+from urlparse import urlparse
+from os.path import splitext,basename
+from multiprocessing import Pool,TimeoutError
 
 class ImagesDownload(object):
 
-	def __init__(self, fileName = None):
+	def __init__(self, pool_size=5,timeout=10,fileName = None):

```

Comment:
I see. or you can benefit from it being a named parameter and pass it with the parameter name, such as: 
ImagesDownload(filename="ImageCollection.txt")